docs/quickstart.rst

```diff
@@ -11,6 +11,14 @@ Install a package from `PyPI`_:
     [...]
     Successfully installed SomePackage
 
+Install a package downloaded from `PyPI`_ or got elsewhere. This is useful if the target machine does not have a network connection:
+
+::
+
+  $ pip install SomePackage.tar.gz
```

Comment:
I'd prefer it if the example was a .whl file, as typically installing wheels is a much better idea. (Obviously users can still install sdists, and the documentation explains how to, but for a quick start using a wheel as an example gets the message across that if you have to download for offline installs, prefer wheels and here's how to install them...